test/test_transform.py

```diff
@@ -898,6 +898,7 @@ def test_AddObservedIndicator():
     array_values = [
         np.array([np.nan, 1.0, 1.0, np.nan, 2.0, np.nan, 1.0, np.nan]),
         np.array([np.nan]),
```

Comment:
So there was a test with all nan, but it also had just one value!

Reply:
Yeah because having a single missing values is another edge case!